### PR TITLE
Hell w app

### PR DESCRIPTION
**Pull request by NewKidlp2**

### CODE REVIEW

**Review by goponenkoev — CHANGES_REQUESTED** [review]
For begin, plz remove all file configured by building project in your ide

**Review by goponenkoev — CHANGES_REQUESTED** [review]
Also remove index.html. It’s unnecessary in our project

File: MyServlet/build.gradle [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        classpath 'org.akhikhl.gretty:gretty:+'
+    }
+}
+
+plugins {
+    id 'java'
+}
+apply plugin: 'war'
+apply plugin: 'org.akhikhl.gretty'
+group 'HelloServlet'
```
Comment (DeryckSPBU) [inline MyServlet/build.gradle:15]: Usually logical blocks are separated with one vertical space (between 12-13 and 14-15 rows)

File: MyServlet/build.gradle [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    testCompile group: 'junit', name: 'junit', version: '4.12'
+}
+
+dependencies {
```
Comment (DeryckSPBU) [inline MyServlet/build.gradle:28]: why do we need two dependencies blocks?

**Comment by DeryckSPBU** [comment]
Also we do not need compiled classes (.class in WEB-INF dir) and external directory "MyServlet", so please remove them

File: MyServlet/build.gradle (hunk 2 above)
Comment (goponenkoev) [reply MyServlet/build.gradle:28]: Yeap, and with two the same dependencies at the first glance

**Review by goponenkoev — CHANGES_REQUESTED** [review]
I think this PR could be closed, 'cause every necessary change already implemented in another one (Authorize form)

**Pull request closed without merging**